integration_tests/tests/consistency/consistency_issue_enhanced.sql

```diff
@@ -5,7 +5,7 @@
 ) }}
 
 {# Exclude columns that depend on calculations involving the current time in seconds or aggregate strings in a random order, as they will differ between runs. #}
-{% set exclude_columns = ['open_duration_seconds', 'any_assignment_duration_seconds', 'last_assignment_duration_seconds', 'fixes_versions', 'affects_versions', 'conversation', 'sprint'] %}
+{% set exclude_columns = ['source_relation', 'open_duration_seconds', 'any_assignment_duration_seconds', 'last_assignment_duration_seconds', 'fixes_versions', 'affects_versions', 'conversation', 'sprint'] + var('consistency_test_exclude_metrics', []) %}
```

Comment:
For ones like this where we're doing a `dbt_utils.star`, could you remove source_relation from here and instead temporarily set it through the `consistency_test_exclude_metrics` variable?

Reply:
removed